src/plugin.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {memo} from 'react'
+
+import Input from './components/Input'
+import {VideoThumbnail} from './components/VideoSource.styled'
+import type {Config, MuxInputProps, VideoAssetDocument} from './util/types'
+
+export function muxVideoCustomRendering(config: Config) {
+  return {
+    components: {
+      input: memo((props: MuxInputProps) => <Input config={config} {...props} />),
```

Comment:
Is memo necessary here?

Reply:
I wasn't sure TBH, I left this in because the FormInput component this was copied from [used it](https://github.com/sanity-io/sanity-plugin-mux-input/blob/145630221069d140025f62defe96dc27fb01b592/src/components/FormInput.tsx#L13) 🤷 .  The FormInput control was using the props information outside of the Input component though, so I guess it made more sense to memo that call where here it doesn't.  I'll remove it 👍